examples/script_nifti.py

```diff
@@ -1,10 +1,37 @@
+import os
+import pathlib
+import shutil
+import sys
 from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
 
 def main():
+    scriptPath = pathlib.Path(__file__).parent.absolute()  # Folder where this script is located
+    dataPath = os.path.join(scriptPath, 'data_testing')  # Folder where the data will be downloaded
+
+    # Download data when not already present
+    if not os.path.isdir(dataPath):
+        url = "https://osf.io/7d2j5/download?versi"
+        try:
+            with ur.urlopen(url) as resp, open("osf_data.zip", 'wb') as out_file:
+                shutil.copyfileobj(resp, out_file)
+                with ZipFile("osf_data.zip", 'r') as zipObj:
+                    # Extract all the contents of zip file in current directory
+                    zipObj.extractall(scriptPath)
+        except IndexError:
+            print("ERROR - {0}:{1}".format(sys.exc_info()[0], sys.exc_info()[1]))
```

Comment:
Don't turn exceptions into prints(). For one thing, it means errors end up on stdout instead of stderr, which mucks up people trying to just extract all (regular, stdout) output from your programs by running `python script.py > output.txt`. The more important reason is that python already does print exceptions when they happen, and moreover it *ends the program* when they happen.

[The Zen of Python] says:

> Errors should never pass silently.
> Unless explicitly silenced.

Just write

```suggestion
        with ur.urlopen(url) as resp, open("osf_data.zip", 'wb') as out_file:
            shutil.copyfileobj(resp, out_file)
            with ZipFile("osf_data.zip", 'r') as zipObj:
                # Extract all the contents of zip file in current directory
                zipObj.extractall(scriptPath)
```

and trust that if the lower levels can't handle something they'll complain about it. It's an important principle in python that programs *don't continue* past errors, unless they have a very good idea that they've handled them.